serverManager/common/include/ISessionServerAppManager.h

```diff
@@ -38,8 +38,10 @@ class ISessionServerAppManager
     ISessionServerAppManager &operator=(const ISessionServerAppManager &) = delete;
     ISessionServerAppManager &operator=(ISessionServerAppManager &&) = delete;
 
-    virtual bool setSessionServerState(const std::string &appId, const service::SessionServerState &newState) = 0;
-    virtual void onSessionServerStateChanged(const std::string &appId, const service::SessionServerState &newState) = 0;
+    virtual bool setSessionServerState(const std::string &appId,
+                                       const firebolt::rialto::common::SessionServerState &newState) = 0;
```

Comment:
In this pull request, you changed SessionServerState (one of rialto::servermanager public enums) to firebolt::rialto::common. Maybe it's worth to change all rialto::servermanager namespace occurences to firebolt::rialto::servermanager? Just a suggestion, what do others think?

Reply:
Yeah that makes sense, will do.